File: app/src/main/java/com/an/github/di/module/ActivityModule.java
package com.an.github.di.module;

import com.an.github.ui.activity.GithubListActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ActivityModule {

    @ContributesAndroidInjector()
    abstract GithubListActivity contributeGithubListActivity();
}

File: app/src/main/java/com/an/github/di/module/ActivityModule.kt
package com.an.github.di.module

import com.an.github.ui.activity.GithubListActivity

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ActivityModule {

    @ContributesAndroidInjector
    internal abstract fun contributeGithubListActivity(): GithubListActivity
}